test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1208,15 +1208,31 @@ describe("UserWallet - Lightning Pay", () => {
             }),
           ).not.toBeInstanceOf(Error)
         }
-
-        const accountRecordA = await getAccountRecordByTestUserRef("A")
+        accountA.contacts = []
+        accountB.contacts = []
+        accountC.contacts = []
+        await AccountsRepository().update(accountA)
+        await AccountsRepository().update(accountB)
+        await AccountsRepository().update(accountC)
```

Comment:
Resetting the contacts array was a good way of not having to move this test outside the for loop 👍🏾 

This is another way to do this by importing `Account` from `"@services/mongoose/utils"` directly. It gets around having to change the domain type.

```suggestion
        await Account.updateMany(
          { _id: { $in: [accountA.id, accountB.id, accountC.id].map(toObjectId) } },
          { $set: { contacts: [] } },
        )
```